src/modules/git_status.rs

```diff
@@ -275,7 +284,13 @@ impl RepoStatus {
 
     fn is_staged(short_status: &str) -> bool {
         // is_index_modified || is_index_added
-        short_status.starts_with('M') || short_status.starts_with('A')
+        short_status.starts_with('M')
+            || short_status.starts_with('A')
+            || short_status.starts_with('T')
```

Comment:
Please add a test for this condition.

Reply:
added test `shows_staged_typechange_with_count()`